src/Sylius/Bundle/ShopBundle/spec/EventListener/ShopUserLogoutHandlerSpec.php

```diff
@@ -40,7 +41,7 @@ function it_clears_cart_session_after_logging_out_and_return_default_handler_res
         ChannelInterface $channel,
         HttpUtils $httpUtils,
         Request $request,
-        Response $response,
```

Comment:
As it seems that it is not used anywhere else, the `use Symfony\Component\HttpFoundation\Response;` should be removed